.github/workflows/ci_jruby.yml

```diff
@@ -36,85 +28,19 @@ jobs:
         with:
           timeout_minutes: 30
           max_attempts: 2
-          command:  bundle exec rake test:env[norails,rails61,rails60]
+          command:  bundle exec rake test:env[norails,rails61]
```

Comment:
From your commits, it looks like you tried to use Rails 7.0 here. Why did you decide to go with Rails 6.1 instead?

Theoretically, [JRuby supports 7.0 as of 9.4.4.0](https://www.reddit.com/r/ruby/comments/z2z7qd/jruby_9400_released_now_supporting_ruby_31_and/), and [version 9.4.5.0 is fixing bugs related to Rails 7.1](https://github.com/jruby/jruby/issues/7982), so support might exist for that version too.